### ISSUE TITLE
Adapt `get_environment` for ToySGD to be consistent with other environments

### ISSUE THREAD

**Issue by Bronzila** [Bronzila/DACORL#22]
Here you can see the difference:
```python
if env_config["type"] == "ToySGD":
        # setup benchmark
        bench = ToySGD2DBenchmark()
        bench.config.cutoff = env_config["num_batches"]
        bench.config.low = env_config["low"]
        bench.config.high = env_config["high"]
        bench.config.function = env_config["function"]
        bench.config.initial_learning_rate = env_config["initial_learning_rate"]
        bench.config.state_version = env_config["state_version"]
        bench.config.reward_version = env_config["reward_version"]
        bench.config.boundary_termination = env_config["boundary_termination"]
        bench.config.seed = env_config["seed"]
        return bench.get_environment()
    if env_config["type"] == "SGD":
        bench = SGDBenchmark(config=env_config)
```
In order to adapt it, we would need to adjust our config file to feature all important information or add it throughout initialization (such as e.g., the seed). Currently it would suffice to simply add a default `instance_set_path`. However, if we change this we should double check that this does not affect reproducibility.
